components/Hero.tsx

```diff
@@ -0,0 +1,37 @@
+import styles from "@/styles/Hero.module.css";
+import { textFont, titleFont } from "@/pages/_app";
+
+const Hero = () => {
+  return (
+    <div className={styles["hero"]}>
+      <div className={styles["hero__section"]}>
+        <div>
+          <div className={`${titleFont.className} heading-medium ${styles["hero__slogan"]}`}>
+            We are a <br />
+            digital agency
+            <br /> from Melbourne.
+          </div>
+          <div className={`${textFont.className} text-xlarge ${styles["hero__txt"]}`}>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
+          </div>
+        </div>
+        <img
+          className={styles["hero__icon"]}
+          aria-label="hero__icon"
+          src="/images/vector.png"
+          alt="hero icon"
+        />
+      </div>
+      <div className={styles["hero__section"]}>
+        <img
+          className={styles["hero__photo"]}
+          aria-label="hero__photo"
+          src="/images/hero.png"
+          alt="hero photo"
+        />
+      </div>
+    </div>
```

Comment:
Semantic HTML Tags please.

Reply:
![image](https://user-images.githubusercontent.com/54622899/217820853-55a59c47-926f-4a59-91bc-33b93552eae0.png)
Does this naming make more sense?

Reply:
It is far better than before. BTW the outer div can be a section as well. It is totally okay to have a section tag inside of another section tag.

However, I think these 2 article tags may need to think again. 

Semantic HTML doesn't mean you can not use DIV.